src/config/project-types.js

```diff
@@ -0,0 +1,54 @@
+// project type configuration
+// used for
+// - create project command args
+// - devServer name mapping
+//
+export default {
+  'create-react-app': {
+    devServer: {
+      taskName: 'start',
+      args: ['run', 'start'],
+      env: {
+        PORT: '$port',
+      },
+    },
+    create: {
+      // not sure if we need that nesting but I think there could be more to configure
+      args: projectPath => [
+        // used for project creation previous getBuildInstructions
+        'create-react-app',
+        projectPath,
+      ],
+    },
+  },
+  gatsby: {
+    devServer: {
+      taskName: 'develop',
+      // gatsby needs -p instead of env for port changing
+      args: ['run', 'develop', '-p', '$port'],
+    },
+    create: {
+      // not sure if we need that nesting but I think there could be more to configure
+      args: projectPath => [
+        // used for project creation previous getBuildInstructions
+        'gatsby',
+        'new',
+        projectPath, // todo replace later with config variables like $projectPath - so we can remove the function. Also check if it's getting complicated.
+      ],
+    },
+  },
+  nextjs: {
+    devServer: {
+      taskName: 'dev',
+      args: ['run', 'dev', '-p', '$port'],
+    },
+    create: {
+      // not sure if we need that nesting but I think there could be more to configure
+      args: projectPath => [
+        // used for project creation previous getBuildInstructions
+        'github:awolf81/create-next-app', // later will be 'create-next-app' --> issue not filed yet

```

Comment:
Perhaps update the comment for `'github:awolf81/create-next-app'` to reference the issue URL, for tracking?